lib/ardis/base_series.rb

```diff
@@ -42,6 +42,7 @@ def initialize( name:             nil,
                     initializer:      nil,
                     inverse_of:       nil,
                     decorator:        nil,
+                    expiration:       nil,
```

Comment:
@snicky 
I'm not sure why putting `expiration` for a series.
I guess ardis is to make it works like ActiveRecord so I don't think it's good to expire data implicitly.
Maybe I mis-understand it.

@jcham 
Do you have any idea?